server/src/main/java/org/opensearch/indices/IndicesModule.java

```diff
@@ -203,6 +203,7 @@ private static Map<String, MetadataFieldMapper.TypeParser> initBuiltInMetadataMa
         builtInMetadataMappers.put(NestedPathFieldMapper.NAME, NestedPathFieldMapper.PARSER);
         builtInMetadataMappers.put(VersionFieldMapper.NAME, VersionFieldMapper.PARSER);
         builtInMetadataMappers.put(SeqNoFieldMapper.NAME, SeqNoFieldMapper.PARSER);
+        builtInMetadataMappers.put(SeqNoFieldMapper.PRIMARY_TERM_NAME, SeqNoFieldMapper.PARSER);
```

Comment:
This is does not look right, the SeqNoFieldMapper is only managing `SeqNoFieldMapper.NAME` field

Reply:
@cwperks you could reference the `SeqNoFieldMapper::SequenceIDFields::emptySeqID()` to extract these additional fields in the `security` plugin.

Reply:
According to the javadoc in [SeqNoFieldMapper](https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/index/mapper/SeqNoFieldMapper.java#L57-L71) is manages both:

```
/**
 * Mapper for the {@code _seq_no} field.
 * <p>
 * We expect to use the seq# for sorting, during collision checking and for
 * doing range searches. Therefore the {@code _seq_no} field is stored both
 * as a numeric doc value and as numeric indexed field.
 * <p>
 * This mapper also manages the primary term field, which has no OpenSearch named
 * equivalent. The primary term is only used during collision after receiving
 * identical seq# values for two document copies. The primary term is stored as
 * a doc value field without being indexed, since it is only intended for use
 * as a key-value lookup.
 *
 * @opensearch.internal
 */
```

What would be a good way of handling this to get it to the security plugin?

I'm also confused about the [SizeFieldMapper](https://github.com/opensearch-project/OpenSearch/blob/main/plugins/mapper-size/src/main/java/org/opensearch/index/mapper/size/SizeFieldMapper.java) as well. I see "_size" being tracked in a list of meta fields from ES before 7.8: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/index/mapper/MapperService.java#L204-L208

Reply:
> What would be a good way of handling this to get it to the security plugin?

`SeqNoFieldMapper::SequenceIDFields::emptySeqID()`

Reply:
Thank you @reta! Closing this PR.

I pushed another commit to the PR in the security plugin where we are trying to extract this list of meta fields: https://github.com/opensearch-project/security/pull/4370

Reply:
`META_FIELDS_BEFORE_7DOT8 ` seems to be never used, could we drop it?